operators/src/processing/temporal_raster_aggregation.rs

```diff
@@ -13,26 +14,33 @@ use geoengine_datatypes::{
     primitives::{SpatialBounded, TimeInstance, TimeInterval, TimeStep},
     raster::{Grid2D, Pixel, RasterTile2D, TileInformation, TilingSpecification},
 };
+use log::debug;
 use serde::{Deserialize, Serialize};
 use snafu::ensure;
 use typetag;
 
-use crate::engine::{ExecutionContext, Operator, RasterOperator, SingleRasterSource};
-
 #[derive(Debug, Clone, Copy, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct TemporalRasterAggregationParameters {
-    aggregation_type: AggregationType,
+    aggregation: Aggregation,
     window: TimeStep,
-    ignore_no_data: bool,
     // TODO: allow specifying window start instead of using query.start?
 }
 
 #[derive(Debug, Clone, Copy, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
-pub enum AggregationType {
-    Min,
-    Max,
+#[serde(tag = "type")]
+pub enum Aggregation {
+    #[serde(rename_all = "camelCase")]
+    Min {
+        ignore_no_data: bool,
+    },
+    #[serde(rename_all = "camelCase")]
+    Max {
+        ignore_no_data: bool,
+    },
+    First,
```

Comment:
Chris said ignore_no_data makes sense for First and Last after all:
- First
  - ignore_no_data = true: takes the first valid value, if there is any
  - ignore_no_data = false: takes the first value, even if it is no_data

there are applications for both cases. What do you think?